lib/src/ruby/api.cc

```diff
@@ -3,8 +3,8 @@
 #include <facter/facts/scalar_value.hpp>
 #include <facter/facts/map_value.hpp>
 #include <facter/facts/array_value.hpp>
-#include <facter/util/environment.hpp>
```

Comment:
It would've simplified your life before... I wouldn't change the references to anything in <facter/...>, as those APIs still need to remain and work the same.